### PR TITLE
wsctl config

### PR DESCRIPTION
**Pull request by huangzhiran**

### CODE REVIEW

File: cmd/wsctl/main.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+func main() {
+	logger := log.Std()
+
+	readConfig, defaultConfigFile, err := config.InitConfig()
+	if err != nil {
+		logger.Panic(err)
+	}
+	client := client.NewClient(readConfig, defaultConfigFile, logger)
+	if err := cmd.NewWsctl(client).Execute(); err != nil {
+		os.Exit(1)
```
Comment (CoderZhi) [inline cmd/wsctl/main.go:21]: print error

File: pkg/wsctl/client/client.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	return c.configFilePath
+}
+
+func (c *client) SelectTranslation(trls map[config.Language]string) string {
+	trl, ok := trls[c.cfg.Language]
+	if ok {
+		return trl
+	}
+	c.logger.Panic(errors.New("failed to pick a translation"))
+	return ""
```
Comment (CoderZhi) [inline pkg/wsctl/client/client.go:50]:
```
  if !ok {
    c.logger.panic...
  }
  return trl
}
```

File: pkg/wsctl/cmd/config/config.go [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+		info.config.Endpoint = _defaultEndpoint
+		completeness = false
+	}
+	if !completeness {
+		if err = info.writeConfig(); err != nil {
+			return info.config, info.configFile, err
+		}
+	}
+	if !isSupportedLanguage(info.config.Language) {
+		fmt.Printf("Warn: Language %s is not supported, English instead.\n", info.config.Language)
```
Comment (CoderZhi) [inline pkg/wsctl/cmd/config/config.go:93]: using English

**Review by CoderZhi — APPROVED** [review]

File: cmd/wsctl/main.go (hunk 1 above)
Comment (huangzhiran) [reply cmd/wsctl/main.go:21]: https://github.com/iotexproject/iotex-core/blob/master/tools/newioctl/ioctl.go#L25

Comment (huangzhiran) [reply cmd/wsctl/main.go:21]: same like ioctl. don't need print err here, each child cmd will print err itself

**Pull request merged**